static/src/stylesheets/module/_layout-hints.scss

```diff
@@ -109,6 +109,11 @@ figure.element-video {
     }
 }
 
+.youtube-media-atom ~ figcaption {
```

Comment:
Why don't you make your own caption class to apply the styling to? like `caption--youtube-atom`